build.gradle.kts

```diff
@@ -142,6 +142,10 @@ tasks.rat {
     // Sample text file for Java quickstart
     "sdks/java/maven-archetypes/examples/sample.txt",
 
+   // Ignore Playground proto/grpc proto/dart generated wrappers
+    "playground/backend/internal/api/v1/api_grpc.pb.go",
+    "playground/frontend/playground_components/lib/src/api/v1/*.dart",
```

Comment:
No need for this because this merge is pending: https://github.com/apache/beam/pull/23378/files#diff-c0dfa6bc7a8685217f70a860145fbdf416d449eaff052fa28352c5cec1a98c06